docfiles/docs.js

```diff
@@ -127,8 +130,6 @@ function setupSidebar() {
     for (var i = 0; i < searchIcons.length; i++) {
         searchIcons.item(i).onkeydown = handleEnterKey;
     }
-
-    scrollActiveHeaderIntoView();
```

Comment:
this was dead code as it was trying to scroll the sidebar when it was hidden by default.